assets/src/blocks/amp-story/index.js

```diff
@@ -121,12 +121,8 @@ export const settings = {
 						</amp-story-grid-layer>
 					)
 				}
-				<amp-story-grid-layer template="fill" style={ overlayStyle }>
```

Comment:
We should probably add this back since that's how the background overlay is displayed :)

Reply:
It's still there..?

Reply:
Oh, right, sorry, didn't notice the green lines below are for `fill` layer.